presentation/src/main/java/com/tdtd/presentation/ui/detail/DetailUserActivity.kt

```diff
@@ -57,4 +60,13 @@ class DetailUserActivity : AppCompatActivity() {
             show(supportFragmentManager, bottomSheet.tag)
         }
     }
+    
+    private fun onClickLeaveRoomButton() {
+        binding.leaveRoomButton.setOnClickListener {
+            val dialog = LeaveRoomDialog.getInstance(submitButtonClicked = {
+                Toast.makeText(applicationContext, "submit!", Toast.LENGTH_SHORT).show()
```

Comment:
요런식으로 사용하시면 됩니다. 메시지도 string.xml에 만들어 놓고 가져다가 사용하시면 좋을 것 같아요.
```suggestion
            this.toast(메시지)
```

Reply:
감사합니다~!